Makefile.Microsoft_nmake

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this makefile specific to MSVC? Which versions of MSVC would it work with?

Reply:
Yes, this is specific to MSVC, and I currently only test with this version, 
```
Microsoft (R) C/C++ Optimizing Compiler Version 19.42.34432 for x86
```

Reply:
Can you leave a comment to this end in the commit message and in the Makefile? (Hopefully adding an older version as well, see other comment)